docs/advanced-chart-features/annotations-and-fibonacci-retracements.md

```diff
@@ -123,4 +123,4 @@ Let’s create the Fibonacci retracements chart with seven labels, seven horizon
         };
     
 
-Check the live demo [here](http://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/annotations/fibonacci-retracements).
+Check the live demo [here](https://jsfiddlefiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/annotations/fibonacci-retracements).
```

Comment:
Ouch, something went wrong with the text replacement with all the jsFiddle linkes  (`http://jsfiddle.net` => `https://jsfiddlefiddle.net`).

Reply:
Should be corrected now.